app-admin/conky/patches/conky-buildfix.patchset

```diff
@@ -0,0 +1,31 @@
+From 015aa63a923aada47bf14b6b315920261705ad61 Mon Sep 17 00:00:00 2001
+From: Andrej Antunovikj <tunas@cryptolab.net>
+Date: Sun, 8 Dec 2019 10:53:32 +0400
+Subject: Haiku build fix: avoid 'share'
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e5e891e..e697569 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -72,7 +72,7 @@ if(BUILD_TESTS)
+ endif()
+ 
+ if(NOT DEFINED DOC_PATH)
+-  set(DOC_PATH "share/doc/${CPACK_PACKAGE_NAME}-${VERSION}")
++  set(DOC_PATH "data/doc/${CPACK_PACKAGE_NAME}-${VERSION}")
+ endif(NOT DEFINED DOC_PATH)
+ set(DOC_FILES extras/convert.lua data/conky_no_x11.conf data/conky.conf)
+ 
+@@ -83,7 +83,7 @@ set(HTML_FILES
+     doc/lua.html
+     doc/variables.html)
+ 
+-set(MAN_PATH "share/man/man1")
++set(MAN_PATH "data/man/man1")
```

Comment:
Here you just want to surround the line with an `if(NOT DEFINED  …)` as with `DOC_PATH`.

Reply:
(and pass `-DMAN_PATH=…`)